Mathlib/Data/Sym/Card.lean

```diff
@@ -5,8 +5,10 @@ Authors: Yaël Dillies, Bhavik Mehta, Huỳnh Trần Khanh, Stuart Presnell
 -/
 import Mathlib.Algebra.BigOperators.Basic
 import Mathlib.Data.Finset.Sym
+import Mathlib.Data.Finsupp.Multiset
 import Mathlib.Data.Fintype.Sum
 import Mathlib.Data.Fintype.Prod
+import Mathlib.SetTheory.Cardinal.Finite
```

Comment:
We could avoid this import if we stated `Nat.card_sum_eq_choose` using `Fintype.card` instead of `Nat.card`. I prefer `Nat.card` but I'm open to dissenting opinions.